tests/unit/setup/4-list-table-split.sql

```diff
@@ -0,0 +1,56 @@
+create table table_1 (
```

Comment:
Here again let's add some comments about what the script intends to test and setup. It's not as obvious as other tests; it's only fair now because of the context of this PR. I believe we will later thank ourselves for adding comments that might seem a tad obvious just now.